### PR TITLE
Add more ES6 tests

### PR DESCRIPTION
**Pull request by garymathews**
- Implement tests for `import` and `export` of modules

### CODE REVIEW

File: Resources/es6.class.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * Copyright (c) 2018 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+import Test from './es6.class.js';
+
+const SUCCESS = 'SUCCESS!';
+
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
```
Comment (sgtcoolguy) [inline Resources/es6.class.test.js]: unnecessary

File: Resources/es6.import.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * Please see the LICENSE included with this distribution for details.
+ */
+import * as a from './es6.module';
+import b from './es6.module';
+import { testExport } from './es6.module';
+
+const SUCCESS = 'SUCCESS!';
+
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
```
Comment (sgtcoolguy) [inline Resources/es6.import.test.js]: unnecessary

File: Resources/app.js [hunk 3]
```diff
@@ -36,13 +36,11 @@ describe('global', function () {
 // ES6 syntax/compatability tests
 require('./es6.arrows.test');
 require('./es6.default.args.test');
-// The JSCore used in Windows doesnt support rest args yet,
-// avoid requiring it until it does otherwise it will crash
-if (!utilities.isWindows()) {
-	require('./es6.rest.args.test');
-}
+require('./es6.rest.args.test');
```
Comment (sgtcoolguy) [inline Resources/app.js]: Is this Ok to merge in now? Does Windows no longer crash with this on master branch?

**Comment by garymathews** [comment]
@sgtcoolguy Updated, okay to merge!

**Pull request merged**